src/btree/bt_stat.c

```diff
@@ -44,6 +44,9 @@ __wt_btree_stat_init(WT_SESSION_IMPL *session, WT_CURSOR_STAT *cst)
 	WT_STAT_SET(session, stats, cache_bytes_inuse,
 	    __wt_btree_bytes_inuse(session));
 
+	if (F_ISSET(cst, WT_CONN_STAT_CACHE_WALK))
+		WT_RET(__wt_curstat_cache_walk(session));
+
 	/* Everything else is really, really expensive. */
```

Comment:
It seems like this comment isn't quite right any more -- or are we saying that a cache walk is "really expensive" but not "really, really expensive"?  ;-)